src/services/shims.ts

```diff
@@ -322,8 +323,19 @@ module ts {
         }
     }
 
+    function serializeResultAsJson(target: Shim, key: string | symbol, descriptor: PropertyDescriptor) {

```

Comment:
I just meant leave PropertyDecorator as it is, and then modify the check that compares the the declared decorator to PropertyDecorator.